easybuild/easyconfigs/i/igraph/igraph-0.7.1-intel-2016b.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'ConfigureMake'
+
+name = 'igraph'
+version = '0.7.1'
+
+homepage = 'http://igraph.org'
+description = """igraph is a collection of network analysis tools with the emphasis on 
+efficiency, portability and ease of use. igraph is open source and free. igraph can be 
+programmed in R, Python and C/C++."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+toolchainopts = {'pic': True}
+
+sources = ['igraph-%(version)s.tar.gz']
+source_urls = ['https://github.com/igraph/igraph/releases/download/%(version)s']
+
+builddependencies = [('Autotools', '20150215')]
+
+patches = [
+	# Author: Maxime Boissonneault <maxime.boissonneault@calculquebec.ca>
+	# Remove explicit linking on -llapack and -llblas, which are not found when linking against MKL
```

Comment:
@mboisson please add this in the actual patch files themselves, any strings above the first `---` are basically comments, you won't break the patch

see for example https://github.com/hpcugent/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/c/Cantera/Cantera-2.3.0_fix-bug-416.patch